js/web/lib/wasm/jsep/backend-webgpu.ts

```diff
@@ -243,13 +243,21 @@ export class WebGpuBackend {
       requiredFeatures,
     };
 
-    if (adapter.features.has('chromium-experimental-timestamp-query-inside-passes')) {
-      requiredFeatures.push('chromium-experimental-timestamp-query-inside-passes' as GPUFeatureName);
-    } else if (adapter.features.has('timestamp-query')) {
-      requiredFeatures.push('timestamp-query');
-    }
-    if (adapter.features.has('shader-f16')) {
-      requiredFeatures.push('shader-f16');
+    // Try requiring WebGPU features
+    const tryRequireFeature = (feature: GPUFeatureName) =>
```

Comment:
Can we just rename it to requireFeature? try do sth is a bit weird.

Reply:
Made it `requireFeatureIfAvailable`.